src/main/java/xbdd/webapp/util/DatabaseUtilities.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2015 Orion Health (Orchestral Development Ltd)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package xbdd.webapp.util;
+
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+
+public final class DatabaseUtilities {
+	private DatabaseUtilities() {

```

Comment:
I've always been told that adding a private constructor to a static utility class is good practice to prevent accidental/mistaken instantiations of the class. You're right, it kinda doesn't do anything so I can get rid of it if you'd prefer. :)